packages/console/src/pages/SignInExperience/index.tsx

```diff
@@ -122,23 +134,28 @@ const SignInExperience = () => {
     <div className={styles.container}>
       <CardTitle title="sign_in_exp.title" subtitle="sign_in_exp.description" />
       <TabNav className={styles.tabs}>
-        <TabNavItem href="/sign-in-experience/branding">
+        <TabNavItem href="/sign-in-experience/branding" errorCount={getBrandingErrorCount(errors)}>
           {t('sign_in_exp.tabs.branding')}
         </TabNavItem>
-        <TabNavItem href="/sign-in-experience/sign-up-and-sign-in">
+        <TabNavItem
+          href="/sign-in-experience/sign-up-and-sign-in"
+          errorCount={getSignUpAndSignInErrorCount(errors)}
+        >
           {t('sign_in_exp.tabs.sign_up_and_sign_in')}
         </TabNavItem>
-        <TabNavItem href="/sign-in-experience/others">{t('sign_in_exp.tabs.others')}</TabNavItem>
+        <TabNavItem href="/sign-in-experience/others" errorCount={getOthersErrorCount(errors)}>
+          {t('sign_in_exp.tabs.others')}
+        </TabNavItem>
       </TabNav>
       <div className={styles.content}>
         <div className={classNames(styles.formWrapper, isDirty && styles.withSubmitActionBar)}>
           {!data && error && <div>{`error occurred: ${error.body?.message ?? error.message}`}</div>}
           {data && defaultFormData && (
             <FormProvider {...methods}>
-              <form id={formId} className={styles.form}>
-                {tab === 'branding' && <Branding />}
-                {tab === 'sign-up-and-sign-in' && <SignUpAndSignIn />}
-                {tab === 'others' && <Others />}
+              <form>
+                <Branding isActive={tab === 'branding'} />
+                <SignUpAndSignIn isActive={tab === 'sign-up-and-sign-in'} />
+                <Others isActive={tab === 'others'} />
```

Comment:
should we use enum to guard those tab variables?

Reply:
These tabs come from the URL.
Since we don't have constants for our router paths, I thought it would be ok to use primitive string values here.

Reply:
Add an issue to track this.
https://linear.app/silverhand/issue/LOG-4766/add-constants-to-guard-router-path